yabt/builders/cpp.py

```diff
@@ -252,6 +259,11 @@ def compile_cc(build_context, compiler_config, buildenv, sources,
        and return list of generated object file.
     """
     objects = []
+    compile_commands = []
```

Comment:
This name isn't clear since we create the compile command on our own (we have `compile_cmd` variable). What does this `compile_commands` include? What is the difference with `compile_cmd`?